### ISSUE TITLE
Loki, connection export->import makes connection with session_logs bound to old user /home 

### ISSUE THREAD

**Issue by grey-olli** [asbru-cm/asbru-cm#561]
**Describe the bug**
Connection import (or export - I donno where you will fix this) should not make session log folder configuration inaccessible after importing a connection from yaml config. 

**To Reproduce**
1. Start as user user0, create connection, enable session logs, choose folder below user0 home.
2. export connection to yaml
3. Import as another linux user1.
4. Start , try to connect
5. The asbru-cm hangs, session logs file is inaccessible since user1 has no rights to user0 home.

**Expected behavior**
inaccessible session log file should:
1) produce a warning on connection save
2) checkbox should be automatically disabled on attempt to use it
3) should NOT hang entire application.
4) Open folder button should give a warning about inaccessibility

asbru-cm may convert /home/otheruser to /home/currentuser when importing connection configuration. 

**Desktop (please complete the following information):**
 - OS: [Linux fedora 31]
[olli@dell15 ~]$ uname -a
Linux dell15.localdomain 5.5.11-200.fc31.x86_64 #1 SMP Mon Mar 23 17:32:43 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
[olli@dell15 ~]$

 - Version [asbru-cm 6.2.0]

**Comment by gfrenoy** [comment]
Thanks for the report.

I made a change for the most important point : it should not hang (and display a warning that your configuration is wrong).

At this stage, I would believe it's enough.  Let me know if it helps.

**Comment by grey-olli** [comment]
Yep, it is sufficient. How long it usually takes to deliver fix via packagecloud.io? I mean this is not blocker & you may close issue since fix is already working at your side - the code looks fine.

**Comment by gfrenoy** [comment]
> How long it usually takes to deliver fix via packagecloud.io?

Typically 15 minutes (after the commit push).

If you have the opportunity to test this and close the issue ; that'd be great and would help the project.

**Comment by grey-olli** [comment]
It doesn't hang now. But if you run asbru-cm from DE GUI menu - the user will be unaware about wrong settings - stderr messages are silently lost and there's no intuitive way to know that something is wrong - GUI shows last folder in path, attempt to open a folder just doesn't show a folder window - this will confuse users. With commercial software I wouldn't close this - this behaviour is not perfect. 
Though the bug has history - if someone 'll catch this - all details are here. You should close.

**Comment by gfrenoy** [comment]
Well, commercial or not ; it's all about the same story : it must offer the best possible user experience !

And you are right, a visual feedback about the issue would help.   Certainly when you open the (faulty) settings !

Now all is question of means and priority... As this is not crashing anymore, we can lower a bit the priority.  We'd like to focus on the "official" release of 6.2 ; so I'd propose to keep this open and schedule it for 6.3.

In the meantime, thanks for your feedback.

And for anybody reading this : don't forget PR's are always welcome :)
